app/views/operations/index.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= stylesheet_link_tag 'operations/index', media: 'all' %>
+<div class="container">
+  <div class="operation__header">
+    <%= link_to groups_path do %>
+      <i class="fas fa-arrow-left"></i>
+    <% end %>
+    <h1 class="operation__title"> Operations </h1>
+    <%= link_to destroy_user_session_path do %>
+      <i class="fas fa-sign-out-alt"></i>
+    <% end %>
+  </div>
+
+  <div class="operations__container">
+
+    <div class="operations__info">
+      <p>Total Amount: </p>
+      <span>$<%= @group.total_spent %></span>
+    </div>
+    <table>
+      <thead>
+        <tr>
+          <td>Operation</td>
+          <td>Date</td>
+          <td>Name</td>
+          <td>Cost</td>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% if @operations.empty? %>
+          <tr>
+            <td>
+              <img src="https://cdn2.iconfinder.com/data/icons/ecommerce-filled-outline-1/32/Ecommerce-shopping-sale-transaction_28-512.png" alt="transaction icon">
+            </td>
+            <td>No operations found</td>
+          </tr>
+        <% end %>
+
+        <% @operations.each do |operation| %>
+          <tr>
+            <td>
+              <img src="https://cdn2.iconfinder.com/data/icons/ecommerce-filled-outline-1/32/Ecommerce-shopping-sale-transaction_28-512.png" alt="transaction icon">
+            </td>
+            <td><%= operation.created_at %></td>
+            <td><%= operation.name %></td>
+            <td><%= operation.amount %></td>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+
+  <div class="add__section">
+    <%= link_to new_group_operation_path do %>
+      <button class="operation__add">Add new operation</button>
+    <% end %>
+  </div>
```

Comment:
- [ ] Based on the design guidelines, the **_Add New Transaction_** button should be at the bottom of the page. I would advise you to implement it this way. It would make your UI better and make your work look identical to that of the template provided. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/227339281-29d3114c-18c7-48e2-afff-7d58416fd94c.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/227339445-5fcef1d3-398f-49c3-a8a5-ad8daa1f441c.png)

### Design Requirement
![image](https://user-images.githubusercontent.com/77106962/227339622-2270ee2b-3d9c-4ee9-b7ac-01f284173957.png)